deploy.ps1

```diff
@@ -163,6 +163,13 @@ $ServiceAccount = $AdminServiceAccount
 
 Write-Host "Service Account: [$ServiceAccount]"
 
+$CloudbuildServiceAccount = "$ProjectNumber@cloudbuild.gserviceaccount.com"

```

Comment:
Referencing the Cloud Build service account here assumes that API has been enabled for the project. As such "cloudbuild" should be added to the list of required APIs on line 85.

Reply:
I added cloudbuild as one of required APIs on line 85. This requirement is also written in Citrix's document.